### ISSUE TITLE
[Screen Reader-Home Page-Blog]: Alt Attribute is not defined for the image showing near Documention link on blog page.

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2648]
**User Experience:** 
Users who depends on Screen Reader will get confuse if Atl attribute is not defined for the interactive images,then information about the image will get missed from users.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Tool Used: IE11 Developer (F12)

**Repro Steps:-**
1. Navigate to https://makecode.com/ 
2. Navigate to the Blog in header section and select it.
3. Navigate to the image showing near Documentation link in left panel.
6. Verify that alt attribute is defined or not for the image.
 
**Actual Result:-**
Alt Attribute is not defined for the image showing near Documention link on blog page.

**Expected Result:-**
Alt Attribute should be defined for the image showing near Documention link on blog page.

**Note:** This issue exists with NVDA +FF.
 
**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

**Suggested Fix:**
Using a text alternative on one item within a group of images that describes all items in the group 
e.g.
<p>Rating: 
         <img src="star1" alt="3 out of 5 stars">
          <img src="star1" alt="">
          < img src="star1" alt="">
           < img src="star2" alt="">
           < img src="star2" alt="">
  </p>

**Please refer the attachment.**
![mas1 1 1_homepage_blog](https://user-images.githubusercontent.com/30434417/28909353-c570203a-7845-11e7-9199-db39d460b6c8.jpg)

**Comment by v-mesala** [comment]
Bug is verified on URL : https://makecode.com/ and bug still exists as Still alt is not defined for the image showing near Documentation.Hence Reactivated the bug and added the Active tag.

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.com/ and issue is fixed now. Hence closing the bug
